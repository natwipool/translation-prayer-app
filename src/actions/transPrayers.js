// SET_TRANS_PRAYER
const setTransPrayer = (transPrayers = {}) => ({
  type: 'SET_TRANS_PRAYER',
  transPrayers
});

export default setTransPrayer;