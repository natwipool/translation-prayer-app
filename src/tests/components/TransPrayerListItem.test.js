import React from 'react';
import { shallow } from 'enzyme';
import TransPrayerListItem from '../../components/TransPrayerListItem';
import transPrayers from '../fixtures/transPrayers';

test('should render TransPrayerListItem correctly', () => {
  const wrapper = shallow(<TransPrayerListItem transPrayerList={transPrayers[0]} />);
  expect(wrapper).toMatchSnapshot();
});