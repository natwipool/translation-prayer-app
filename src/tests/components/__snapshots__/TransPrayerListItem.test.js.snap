// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TransPrayerListItem correctly 1`] = `
<div>
  <h3 />
  <p />
</div>
`;
